proto/viam/app/v1/billing.proto

```diff
@@ -12,8 +12,15 @@ service BillingService {
   rpc GetInvoiceHistory(GetInvoiceHistoryRequest) returns (GetInvoiceHistoryResponse);
   rpc GetItemizedInvoice(GetItemizedInvoiceRequest) returns (GetItemizedInvoiceResponse);
   rpc GetBillingSummary(GetBillingSummaryRequest) returns (GetBillingSummaryResponse);
+  // Detailed breakdown of current month's costs
+  rpc GetCurrentMonthUsage(GetCurrentMonthUsageRequest) returns (GetCurrentMonthUsageResponse);
+  // Org-level information (like billing email and payment details)
+  rpc GetBillingInformation(GetBillingInformationRequest) returns (GetBillingInformationResponse);
+  // Total outstanding balance and previous invoices
+  rpc GetBalanceSummary(GetBalanceSummaryRequest) returns (GetBalanceSummaryResponse);
```

Comment:
Amusingly, I originally had it as `GetInvoiceSummary`, however, that was confusing because it meant that `GetInvoiceSummary` returned a list of `InvoiceSummary` objects. I am happy to put it back as `GetInvoiceSummary`, however there is some naming confusion there.

Reply:
GetInvoicesSummary ? making it plural makes it clear a list of them

Reply:
I am happy with that. Changed.